specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4299,40 +4315,50 @@
 		}],
 		"properties": {
 			"defaultLanguageCode": {
-				"type": "string",
-				"default": "en",
+				"$ref": "#/definitions/ImageAnalysisSkillSupportedLanguageCodes",
 				"description": "A value indicating which language code. Default is en."
```

Comment:
"which language code to use"? The sentence seems to end abruptly

Reply:
Yes. Added "to use" at the end. Thanks.